src/chains/eos_1_8/templates/chainActions/account_create.ts

```diff
@@ -122,14 +122,16 @@ export const decomposeAction = (action: EosActionStruct): EosDecomposeReturn =>
     const returnData: Partial<createAccountNativeParams> = {
       accountName: toEosEntityName(data.name),
       creatorAccountName: toEosEntityName(data.creator),
-      creatorPermission: toEosEntityNameOrNull(auth.permission),
+      creatorPermission: toEosEntityNameOrNull(auth?.permission),
       publicKeyActive: toEosPublicKeyOrNull(ownerKey?.key),
       publicKeyOwner: toEosPublicKeyOrNull(activeKey?.key),
     }
+    const partial = !auth?.permission
```

Comment:
to calculate partial, use the resulting value not the source - this way it's consistent and will support any other converter functions we might use in the future i.e. 
const partial = !returnData.creatorPermission || !returnData.publicKeyActive || returnData.publicKeyOwner